src/main/resources/db/migration/V1__create_schema.sql

```diff
@@ -0,0 +1,67 @@
+CREATE TABLE users
+(
+    id       SERIAL PRIMARY KEY,
```

Comment:
```suggestion
    id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
```

You can read more about this [here](https://www.2ndquadrant.com/en/blog/postgresql-10-identity-columns/).